dashboard/utils.py

```diff
@@ -371,6 +370,18 @@ def get_pearson_exam_status(self):  # pylint: disable=too-many-return-statements
             )
             return ExamProfile.PROFILE_INVALID
 
+    def course_has_exam(self, course_run):
+        """
+        Check if the user is authorized for an exam for a course run
+        """
+        return bool(
+            course_run and
+            course_run.edx_course_key and
+            course_run.course and
+            course_run.course.exam_module and
+            self.program.exam_series_code
+        )
```

Comment:
yes it is not part of `CourseRun`, but for course to have exam its program must have  exam_series_code.